packages/app/cypress/e2e/runner/ct-framework-errors.cy.ts

```diff
@@ -231,8 +231,8 @@ describe('Vue', {
 
     verify('sync error', {
       fileName: 'Errors.vue',
-      line: 24,
-      column: 15,
+      line: 34,
+      column: 13,
```

Comment:
These lines changed because the Vue errors file was linted, so these errors are being thrown in a different location now.